README.md

```diff
@@ -1,3 +1,45 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/awstpr.svg?&style=shield&circle-token=752d6ec42799fb9fd56dcf13c142d2f675b0b91f)](https://circleci.com/gh/giantswarm/awstpr)
 
 # awstpr
+
+Specification of the third party objects used to deploy Kubernetes on top of AWS by the "undercloud" Kubernetes running
+[aws-operator](https://github.com/giantswarm/aws-operator).
+
+## Getting Project
+
+Download the latest release: https://github.com/giantswarm/awstpr/releases/latest
```

Comment:
Yeah, but I expect that to be on git tags, not github releases. We can change this to:

```
Check out the latest tag: https://github.com/giantswarm/awstpr/tags
```